pkg/app/master/inspectors/container/probes/http/crawler.go

```diff
@@ -20,11 +20,15 @@ const (
 )
 
 func (p *CustomProbe) crawl(proto, domain, addr string) {
-	p.workers.Add(1)
```

Comment:
why is it removed?

Reply:
This was moved below so the waitgroup doesn’t deadlock if there’s a setup error.